src/main/kotlin/models/SkillType.kt

```diff
@@ -0,0 +1,5 @@
+package models
+
+enum class SkillType {
+    SOFT, HARD
```

Comment:
Why not use this chance to understand the differences between sealed class and enum class and use them in your next mission? 🙂

https://kotlinlang.org/docs/sealed-classes.html

```kotlin
sealed interface Skill {
    data class Soft : Skill
    data class Hard : Skill
}
```